qsharp.html.markdown

```diff
@@ -4,47 +4,49 @@ contributors:
     - ["Vincent van Wingerden", "https://github.com/vivanwin"]
     - ["Mariia Mykhailova", "https://github.com/tcNickolas"]
     - ["Andrew Ryan Davis", "https://github.com/AndrewDavis1191"]
+    - ["Alex Hansen", "https://github.com/sezna"]
 filename: LearnQSharp.qs
 ---
 
 Q# is a high-level domain-specific language which enables developers to write quantum algorithms. Q# programs can be executed on a quantum simulator running on a classical computer and (in future) on quantum computers.
 
-```c#
+```qsharp
```

Comment:
`qsharp` is not supported by Pygments (the syntax highlighter used on Learn X in Y Minutes)

https://pygments.org/languages/

so either disable syntax highlighting by not including any language name here, or stick with C# or whatever is the closest matching language